.gitignore

```diff
@@ -0,0 +1,3 @@
+**/policygen
+**/tfengine
```

Comment:
From what I remember this would cause changes in the tfengine cmd/internal folders to not get picked up. Is that still the case?

Reply:
Let me try

Reply:
```shell
$ go build ./cmd/tfengine/
$ cp tfengine cmd/tfengine/
$ cp tfengine internal/tfengine/
$ git status
On branch gitignore-binaries
Your branch is up to date with 'origin/gitignore-binaries'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   cmd/tfengine/README.md
	modified:   internal/tfengine/load.go
```
Seems to work

Reply:
Note .gitignore is not retroactive, for existing directories that have been tracked by git, it will not be affected. Can you try creating some new directories with name `tfengine`?

Reply:
It does ignore new directories, and new files created under new directories.

I just pushed a commit to **not** ignore directories with those names.